js/api-usage_inference-session/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "api-usage_inference-session",
+  "private": true,
+  "version": "1.0.0",
+  "description": "This example is a demonstration of basic usage of InferenceSession.",
+  "main": "index.js",
+  "dependencies": {
+    "onnxruntime-node": "1.7.0-dev.0"
```

Comment:
^1.7.0?

Reply:
I updated them to `1.8.0`, but they will not work until 1.8.0 release.